### PR TITLE
Tag nightlies after a successful build

### PR DESCRIPTION
**Pull request by stefanceriu**

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -238,17 +235,46 @@ lane :prepare_next_release do
 
   xcodegen(spec: "project.yml")
 
-  sh("git config --global user.name 'Element CI'")
-  sh("git config --global user.email 'ci@element.io'")
```
Comment (pixlwave) [inline fastlane/Fastfile:242]: Wouldn't we want these set before committing so that the commit is shown as made by CI?

File: fastlane/Fastfile [hunk 2]
```diff
@@ -238,17 +235,46 @@ lane :prepare_next_release do
...
+
+lane :tag_nightly do |options|
+  build_number = options[:build_number]
+  UI.user_error!("Invalid build number.") unless !build_number.to_s.empty?
+
+  xcodegen_project_file_path = "../project.yml"
+  data = YAML.load_file xcodegen_project_file_path
+  current_version = data["settings"]["MARKETING_VERSION"]
+
+  tag_name = "nightly-#{current_version}.#{build_number}"
```
Comment (pixlwave) [inline fastlane/Fastfile:253]: Maybe this should be `nightly/versoionnumber` so they're more easily hidden in a git client?

**Review by pixlwave — APPROVED** [review]

File: fastlane/Fastfile (hunk 1 above)
Comment (stefanceriu) [reply fastlane/Fastfile:242]: Yeah, good catch 👍

File: fastlane/Fastfile (hunk 2 above)
Comment (stefanceriu) [reply fastlane/Fastfile]: Yep, makes perfect sense

**Pull request merged**